README.md

```diff
@@ -763,64 +763,45 @@ $ composer require google/cloud-oslogin
 ```php
 require_once __DIR__ . '/vendor/autoload.php';
 
-use Google\Cloud\Tasks\V2beta2\CloudTasksClient;
-use Google\Cloud\Tasks\V2beta2\LeaseDuration;
-use Google\Cloud\Tasks\V2beta2\PullMessage;
-use Google\Cloud\Tasks\V2beta2\PullTarget;
-use Google\Cloud\Tasks\V2beta2\Queue;
-use Google\Cloud\Tasks\V2beta2\Task;
-use Google\Cloud\Tasks\V2beta2\Task_View;
-use Google\Protobuf\Duration;
-
+use Google\Cloud\Tasks\V2beta3\AppEngineHttpQueue;
+use Google\Cloud\Tasks\V2beta3\CloudTasksClient;
+use Google\Cloud\Tasks\V2beta3\Queue;
```

Comment:
Just a small nit, we should add whitespace between this and the client instantiation :) (applies to Tasks/README.md as well)

Reply:
Thanks! https://github.com/googleapis/google-cloud-php/pull/1356